draft-ietf-core-new-block.xml

```diff
@@ -875,15 +881,15 @@
 
       <figure align="center" anchor="cddl"
               title="Structure of the Missing Blocks Payload">
-        <artwork align="center"><![CDATA[; A notional array, the elements of which are to be used
+        <artwork align="center"><![CDATA[; This defines an array, the elements of which are to be used
 ; in a CBOR Sequence:
 payload = [+ missing-block-number]
 ; A unique block number not received:
 missing-block-number = uint
 ]]></artwork>
       </figure>
 
-      <t></t>
+      <t>This CDDL syntax MUST be followed.</t>
```

Comment:
If we go that way, we need to list CDDL as normative.

Reply:
Made RFC8610 normative.